coding-concepts.js

```diff
@@ -8,48 +8,47 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
-
-// a) Your answer:
-// b) Verify and explain:
+//console.log(colors.push("indigo"))
 
+// a) Your answer: will display indigo at the front spot of array
+// b) Verify and explain: It did push indigo into array but displayed the number of elements in the array
 // --------------------1) What will this log?
 
-const cohort = "LEARN 2023"
-// console.log(cohort.length)
+//const cohort = "LEARN 2023"
+ //console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: should log the length of the characters in the string which is 9
+// b) Verify and explain: so it logged 10 bc it is a string and not an array or else it would have been 0 indexed
```

Comment:
Both strings and arrays are zero-indexed.  Computers are actually zero-indexed so indexes will always start at zero.  The .length property counts all characters, including spaces.  :)